de.metas.adempiere.adempiere/base/src/main/java/de/metas/location/geocoding/GeocodingConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.location.geocoding;
+
+import de.metas.location.geocoding.provider.GeocodingProviderName;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@Builder
+public class GeocodingConfig
```

Comment:
I * think* I would have made GeocodingConfig an interface with just the getProviderName method.
But this way it's also OK for me..but might be something to consider when/if we get the 3rd provider

Reply:
cannot guarantee i'll remember this advice until next time 😊